src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -698,4 +750,89 @@ public AuthRequest exchangeCode(String code, String redirectUri) {
         request.addParameter("redirect_uri", redirectUri);
         return request;
     }
+
+    /**
+     * Create a request to send an email containing a link or a code to begin authentication with Passwordless connections.
+     *
+     * <pre>
+     * {@code
+     * AuthAPI auth = new AuthAPI("me.auth0.com", "B3c6RYhk1v9SbIJcRIOwu62gIUGsnze", "2679NfkaBn62e6w5E8zNEzjr-yWfkaBne");
+     * try {
+     *      PasswordlessEmailResponse result = auth.startPasswordlessEmailFlow("user@domain.com", PasswordlessEmailConnection.CODE)
+     *          .execute();
+     * } catch (Auth0Exception e) {
+     *      // Something happened
+     * }
+     * }
+     * </pre>
+     *
+     * @param email the email address to send the code or link to. Must not be null.
+     * @param connection the type of the passwordless email request. Must not be null.
+     *
+     * @return a Request to configure and execute.
+     *
+     * @see <a href="https://auth0.com/docs/connections/passwordless/guides/email-otp">Passwordless Authentication with Email documentation</a>
+     * @see <a href="https://auth0.com/docs/api/authentication#get-code-or-link">Get code or link API reference documentation</a>
+     */
+    public CustomRequest<PasswordlessEmailResponse> startPasswordlessEmailFlow(String email, PasswordlessEmailConnection connection) {
```

Comment:
I still find it a bit misleading the class name "PasswordlessEmailConnection". Even the param name "connection". This is not a connection. The API defines it as "send" or "what to send to the user". And values can be link or code. I think using something like "mode" or "type" makes more sense. If we use connection it could be interpreted as "realm", and that's not what this is.

Reply:
Sure, I'll change it back to `PasswordlessEmailType` as it was before. I changed it here based on an earlier review comment, but I Iagree that `connection` is misleading, even though that is the parameter name in the API.